doc/whats_new.rst

```diff
@@ -153,11 +153,14 @@ Enhancements
 Bug fixes
 .........
 
+   - Fixed a bug where :class:`sklearn.ensemble.AdaBoostClassifier` throws
+     ``ZeroDivisionError`` while fitting data with single class labels.
```

Comment:
Beware this is a rst file, rst and markdown do not have the same syntax. I changed the single back-quotes in double back-quotes.